person.rb

```diff
@@ -0,0 +1,25 @@
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(name: 'Unknown', age: 0, parent_permission: true)
```

Comment:
- [Optional] Hi @mryeminthu, default value for the age parameter was not given. Let's kindly get rid of it since we required each Person to provide age value

Reply:
@FrederickMih Thank you so much, sir